pyproject.toml

```
@@ -41,47 +42,31 @@ test = [
     "pytest-cov",
     "pytest>=6.0",
 ]
-
 [tool.hatch.version]
 source = "nodejs"
 
-[tool.hatch.build.hooks.version]
-path = "ipylgbst/_version.py"
-
 [tool.hatch.metadata.hooks.nodejs]
-fields = ["description", "authors", "urls"]
+fields = ["description", "authors", "urls", "keywords"]
 
-[tool.hatch.build]
-artifacts = [
-    "ipylgbst/nbextension/index.*",
-    "ipylgbst/labextension/*.tgz",
-    "ipylgbst/labextension",
-]
+[tool.hatch.build.targets.sdist]
+artifacts = ["ipylgbst/labextension"]
+exclude = [".github", "binder"]
 
 [tool.hatch.build.targets.wheel.shared-data]
-"ipylgbst/nbextension" = "share/jupyter/nbextensions/ipylgbst"
 "ipylgbst/labextension" = "share/jupyter/labextensions/ipylgbst"
```

```suggestion
"ipylgbst/nbextension" = "share/jupyter/nbextensions/ipylgbst"
"ipylgbst/labextension" = "share/jupyter/labextensions/ipylgbst"
```

I assume the configuration was eliminated by the script to migrate to `V4`, but it's porbably still be needed. There are other extensions who use it and are using the latest version of `JLab`, so it doesn't seem that its removal is needed.